morango/controller.py

```diff
@@ -15,7 +17,7 @@ def __init__(self, profile):
         assert profile, "profile needs to be defined."
         self.profile = profile
 
-    def serialize_into_store(self):
+    def serialize_into_store(self, filters=None):
```

Comment:
Perhaps nice to have this argument be an actual `Filter` class instance.

Reply:
If you don't think that makes sense, then better to rename arg to `filter_prefixes`

Reply:
Ah, nevermind, it *is* a `Filter`, but filters are iterable. I think just `filter` is ideal as the param name here, then.